microservices/CFDP/app/models/cfdp_pdu.rb

```diff
@@ -159,14 +173,21 @@ def self.build_initial_pdu(type:, destination_entity:, file_size:, segmentation_
     else
       pdu.write("CRC_FLAG", "CRC_NOT_PRESENT")
     end
-    if file_size >= 4_294_967_296
+
+    # Version 0 doesn't support large files
+    if version == 0 or file_size < 4_294_967_296
+      pdu.write("LARGE_FILE_FLAG", "SMALL_FILE")  # Equals 0
```

Comment:
version 0 doesn't have the LARGE_FILE_FLAG key if I look above at the keys array. Should this simply be removed and only set this key in the non-version 0?

Reply:
The field still exists in the packet it just effectively got renamed from "RESERVED".  So this is just filling in 0 for version 0.